protocol/x/ratelimit/types/params.go

```diff
@@ -57,7 +57,7 @@ func (p *LimitParams) Validate() error {
 			return ErrInvalidBaselineMinimum
 		}
 
-		if limiter.BaselineTvlPpm == 0 || limiter.BaselineTvlPpm >= lib.OneMillion {
+		if limiter.BaselineTvlPpm == 0 || limiter.BaselineTvlPpm > lib.OneMillion {
 			return ErrInvalidBaselineTvlPpm
```

Comment:
suggestion: return `limiter.BaselineTvlPpm` as part of the error?

Reply:
Seems like the other errors don't return the actual values either. will defer this change to @roy-dydx 🙏